atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -348,7 +348,10 @@ def _validate_platforms_in_image(self, image):
 
         self.log.info('Manifest list arches: %s, expected arches: %s',
                       manifest_list_arches, expected_arches)
-        assert manifest_list_arches >= expected_arches, \
-            'Missing arches in manifest list for base image'
+
+        missing_arches = expected_arches - manifest_list_arches
+        if missing_arches:
+            raise RuntimeError('No images found for arches {} for base image {}'
```

Comment:
probably better message : 
Base image {} isn't available for arches {}

Reply:
That does sound better, thanks